crimemapper/graph_calcs.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+
+"""Graph calculations for graph view."""
+from crimemapper.models import (
+    DBSession,
+    Entry,
+)
+from collections import Counter
+import random
+
+
+UPPER_DICT = {
+    "Assault": ["ASSAULT"],
+    "Theft": [
+        "BIKE THEFT", "BURGLARY", "CAR PROWL", "MAIL THEFT", "PICKPOCKET",
+        "PURSE SNATCH", "SHOPLIFTING", "THEFT OF SERVICE", "VEHICLE THEFT",
+        "BURGLARY-SECURE PARKING-RES"
+    ],
+    "Sin": ["GAMBLE", "PORNOGRAPHY", "PROSTITUTION"],
+    "Drugs": [
+        "DUI", "LIQUOR VIOLATIONS", "NARCOTICS", "STAY OUT OF AREA OF DRUGS"
+    ],
+    "Fraud": ["COUNTERFEIT", "EMBEZZLE", "FORGERY", "FRAUD"],
+    "Homicide": ["HOMICIDE"],
+    "Property": [
+        "ILLEGAL DUMPING", "LOST PROPERTY", "OTHER PROPERTY",
+        "PROPERTY DAMAGE", "RECKLESS BURNING", "RECOVERED PROPERTY",
+        "STOLEN PROPERTY",
+    ],
+    "Robbery": ["ROBBERY"],
+    "Weapons": ["WEAPONS"],
+    "Disturbances": [
+        "DISORDERLY CONDUCT", "DISPUTE", "FIREWORK", "PUBLIC NUISANCE",
+        "THREATS", "TRESPASS"
+    ],
+    "Other": [
+        "ANIMAL COMPLAINT", "BIAS INCIDENT", "ELUDING", "ESCAPE", "EXTORTION",
+        "HARBOR CALLS", "INJURY", "LOITERING", "METRO", "OBSTRUCT", "TRAFFIC",
+        "WARRANT ARREST", "VIOLATION OF COURT ORDER"
+    ],
+}
```

Comment:
It feels to me like this whole `UPPER_DICT` is a duplicate of the exact data that is in the `CRIME_DICT` in `crimedict.py`.  I wonder why we need the duplication.